tests/unit/h/views/account_signup_test.py

```diff
@@ -1,19 +1,18 @@
+from datetime import UTC
```

Comment:
There's only a few small changes in the tests. Most of the diff is just replacing `controller` / `SignupController` with `views` / `SignupViews`.

The next PR completely replaces these tests anyway so you may not want to review these changes too closely.